src/runtime/loader.ts

```diff
@@ -68,14 +68,23 @@ export class Loader {
     return this.loadFile(file, 'utf-8') as Promise<string>;
   }
 
-  async loadBinary(file: string): Promise<ArrayBuffer> {
-    if (/^https?:\/\//.test(file)) {
-      return fetch(file).then(res => res.arrayBuffer());
+  async loadWasmBinary(spec): Promise<ArrayBuffer> {
+    // TODO: use spec.implBlobUrl if present?
+    this.mapParticleUrl(spec.implFile);
```

Comment:
random idea:
```
const mapped = this.mapParticleUrl(spec.implFile);
...
mapParticleUrl(path: string) {
  return path;
}```

Reply:
`mapParticleUrl` doesn't return anything - it's adding to the internal url map to allow `$` macros based on the particle name and location.